addons/medical_ai/StateMachine.hpp

```diff
@@ -1,5 +1,5 @@
 class GVAR(stateMachine) {
-    list = "allUnits select {local _x}";
+    list = "allUnits select {local _x && {_x != ACE_player} && {!isPlayer _x}}";
```

Comment:
If it's not a player then it's also not ACE_player. Are both conditions really necessary?